### DISCUSSION TITLE
Running on gnome

### DISCUSSION THREAD

**Discussion by 4uT0Gen** [Almamu/linux-wallpaperengine#133] [Q&A]
Hi, I wonder if this work on Ubuntu gnome?

**Comment by Almamu** [comment]
Support for desktop environments like GNOME is lacking as these tend to draw on the screen's background, so you won't be able to see it. It's something I have on my radar, but not proritary for now...

**Reply by michas79de** [reply]
Any news?

**Reply by Sprusrroot** [reply]
Any news?
